api/controllers/project.js

```diff
@@ -186,6 +186,8 @@ exports.protectedGet = async function (args, res, next) {
 
   var commentPeriodPipeline = null;
 
+  console.log('The user sub', args.swagger.params.auth_payload.sub)
```

Comment:
@Baelx Not sure this is suppose to be here??

Reply:
I did intentionally leave this in for future testing.